include/cantera/thermo/ThermoPhase.h

```diff
@@ -975,6 +974,118 @@ class ThermoPhase : public Phase
      */
     virtual void setState_SV(doublereal s, doublereal v, doublereal tol = 1.e-4);
 
+    //! Set the specific entropy (J/kg/K) and temperature (K).
+    /*!
+     * This function fixes the internal state of the phase so that the specific
+     * entropy and temperature have the value of the input parameters.
+     * It must be implemented in a derived class.
```

Comment:
It is there to indicate that this method will throw a NotImplemented error if it is used without implementing it. That phrasing is present on a bunch of the similar functions. Is there an alternative phrasing you would prefer?